info/src/main/java/edu/artic/info/InfoModule.kt

```diff
@@ -77,20 +77,32 @@ abstract class InfoModule {
         fun provideRetrofit(@Named(ApiModule.BLOB_CLIENT_API)
                             client: OkHttpClient,
                             executor: Executor)
-                : Retrofit = constructRetrofit(
-                baseUrl = BuildConfig.MEMBER_INFO_API,
-                client = client,
-                executor = executor,
-                customConfiguration = {
-                    addConverterFactory(SimpleXmlConverterFactory.createNonStrict(
-                            Persister(AnnotationStrategy())
-                    ))
-                })
+                : Retrofit? {
+            return if (BuildConfig.MEMBER_INFO_API.indexOf("https") < 0) {
```

Comment:
This works fine but using `contains` would make this more readable IMO.

Reply:
Oh, good point. Let me do that.